tests/test_session.py

```diff
@@ -97,6 +97,12 @@ async def test_connect(
         f"mowers/{MOWER_ID}/calendar",
         json={"data": {"type": "calendar", "attributes": {"tasks": task_list}}},
     )
+    await automower_api.error_confirm(MOWER_ID)
+    assert mocked_method.call_count == 10
+    mocked_method.assert_called_with(
+        f"mowers/{mower_id}/errors/confirm",
```

Comment:
You must use the `MOWER_ID` constant. In capital letters.

Reply:
Yeah I just saw...fixing